ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-openshift-logging-5.5.yaml

```diff
@@ -0,0 +1,90 @@
+base_images:
+  ansible:
+    name: "4.11"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.11"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.11"
+  tools:
+    name: "4.11"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.11"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: azure-ipi-fips-p1-f3-openshift-logging-5-5
```

Comment:
Shall we just add the test into existing profiles `openshift-openshift-tests-private-release-4.xxx__amd64-nightly.yaml` as we do for hypershift/rosa/osd/...

Reply:
We have tied the logging versions to the corresponding openshift-tests-private branches, where Logging 5.7 is tied to release-4.13 and Logging 5.6 is tied to release-4.12. To ensure consistency, we have added new configuration files. Even if we need to add tests to existing profiles, creating new config files for specifying a different openshift-tests-private branch for a particular Logging version running on different OCP versions is necessary. With separate configurations, maintenance becomes easier for us.

Reply:
Can we override the tests-private and tests-private-ui images when adding the tests? For example for running Logging 5.7 tests, we would need tag 4.13 when added to the profiles using different OCP versions as you mentioned. openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml, openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml and openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml,

  tests-private:
    name: tests-private
    namespace: ci
    tag: "4.13"
  tests-private-ui:
    name: tests-private-ui
    namespace: ci
    tag: "4.13"

Reply:
I do not see any other issue that prevents us from adding the Logging tests to existing profile configs. Its just the tests-private branch image that we need to override.

Reply:
In `openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml`, tests-private and tests-private-ui is already 4.13, refer to https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml#L22-L37
,  so I think there is no need to override

Reply:
I mean in openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml, openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml, is it possible to override the tests-private tag?

Reply:
If you check the `zz_generated_metadata` part, you can see each file/test is related to proper branch. E.g, for 4.13,
```
zz_generated_metadata:
  branch: release-4.13
  org: openshift
  repo: openshift-tests-private
  variant: amd64-nightly
```
May i know the reason why do you want to use 4.13 branch to do tests on 4.11/4.12 ?

Reply:
Our Logging version tests code is dependent on the openshift-tests-private branch. The tests for running Logging 5.7 is in release-4.13 branch of the openshift-tests-private repo. Thus if we want to run Logging 5.7 on any of the OCP versions(4.11,4.12,4.13,4.14), we would need to use the release-4.13 branch of the openshif-tests-private repo. Unfortunately the decision for tying Logging versions to tests-private release branches was done when we initially started switching to using ginkgo for our test runs. cc @anpingli @QiaolingTang

Reply:
BTW these tests are for full functionality testing for Logging and will be used for Logging releases.

Reply:
@liangxia  
Logging 5.5 cases are in branch release-4.11, Logging 5.6 cases are in branch release-4.12, Logging 5.7 cases are in branch release-4.13.
So if we want to test Logging 5.5 on OCP 4.13.  Our team must use release-4.11.

Reply:
These profiles are used to run logging function cases, which can not be ran in parallel. and all logging case will take about 5 hours.  I don't think these cases can be executed using existing profiles.

Reply:
Need to use 4.13 images to test 4.11 sounds like a design issue.
Anyway, I'm unholding the PR for now. Leave it to your feature team to decide.